src/transformers/models/auto/configuration_auto.py

```diff
@@ -236,6 +238,7 @@
         ("maskformer", "MaskFormer"),
         ("poolformer", "PoolFormer"),
         ("convnext", "ConvNext"),
+        ("cvt", "Cvt"),
```

Comment:
```suggestion
        ("cvt", "CvT"),
```
You can run `make fix-copies` again after this.